crux-http-server/src/crux/http_server.clj

```diff
@@ -260,6 +265,9 @@
     [#"^/document/.+$" [:get :post]]
     (document crux-node request)
 
+    [#"^/documents" [:post]]
```

Comment:
I'm not sure we should bother right now, but as briefly discussed, a set of documents is a special case, like `/document` with filter, and `/documents/abcd` which is a specific resource. There are various opinions about how to model this in REST idiomatically.